### ISSUE TITLE
unicase no longer builds on stable/beta

### ISSUE THREAD

**Issue by gwillen** [seanmonstar/unicase#9]
Currently errors out with:

```
   Compiling unicase v1.1.0
.../unicase-1.1.0/src/lib.rs:55:20: 55:36 error: use of unstable library feature 'iter_order': needs review and revision (see issue #27737)
.../unicase-1.1.0/src/lib.rs:55         self_chars.cmp(other_chars)
```

**Comment by seanmonstar** [comment]
It requires a feature in 1.5, which is the newest stable as of this week.

**Comment by flosse** [comment]
...but as far a I know a minor update shouldn't break compilation. Am I wrong? Or don't you care about semver?

**Comment by gwillen** [comment]
Ahh, well I didn't realize stable had bumped to 1.5, so that fixes my problem, anyway. I'll leave you folks to discuss other implications. Thanks!

**Comment by seanmonstar** [comment]
@flosse I do care about semver. I think incrementing the major version every time a library uses a newly stabilized core feature is probably not the right direction to take. So, I've published v1.1.1, which currently uses a build script to detect if the installed `rustc` is at least 1.5, and if so then include this new feature.

I'd prefer that detecting if a feature is stabilized be part of the core rustc, but it will need to go through the RFC process, and in the meantime, this fixes it immediately.
